src/api-service/__app__/onefuzzlib/job_templates/default_templates.py

```diff
@@ -0,0 +1,519 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from uuid import UUID
+
+from onefuzztypes.enums import (
+    ContainerType,
+    TaskType,
+    UserFieldOperation,
+    UserFieldType,
+)
+from onefuzztypes.job_templates import JobTemplate, UserField, UserFieldLocation
+from onefuzztypes.models import (
+    JobConfig,
+    TaskConfig,
+    TaskContainers,
+    TaskDetails,
+    TaskPool,
+)
+from onefuzztypes.primitives import Container, PoolName
+
+POOL_HELP = "Execute the task on the specified pool"
+DURATION_HELP = "Number of hours to execute the task"
+TARGET_EXE_HELP = "Path to the target executable"
+TARGET_OPTIONS_HELP = "Command line options for the target"
+VM_COUNT_HELP = "Number of VMs to use for fuzzing"
+RETRY_COUNT_HELP = "Number of times to retry a crash to verify reproducability"
+REBOOT_HELP = "After executing the setup script, reboot the VM"
+TAGS_HELP = "User provided metadata for the tasks"
+
+TEMPLATES = {
+    "afl": JobTemplate(
```

Comment:
Can we maybe split these into individual files, and expose them in the `TEMPLATES` dict separately?